### PR TITLE
Some additions and fixes

### PR DESCRIPTION
**Pull request by BobbyWibowo**
* Added safe.fiery.me support, nicknamed fiery.

* Added -t option to set token for fiery host (associating uploads to specific account, optional of course).

* Recovered timestamp naming when using -p option.

* Re-ordered options positioning with getopts.

* Refactored all extraneous H1 instances in README to H2.

* Matched "Usage" section in README with the latest state in the script's help message.

### CODE REVIEW

**Comment by BobbyWibowo** [comment]
So, I was thinking that I should probably get rid of `-t` option from this, and keep the script as a simple uploader.

If user wants extra features for my host (or [lolisafe](https://github.com/WeebDev/lolisafe)-based hosts in general), they can use my fork instead? It adds the `-t` option, as well as a new one named `-a`, with which you can add uploads to specific "album".
If you're fine with keeping the `-t` option though, I'd very much like to request permission to add `-a` option too.

A little more info about `-a` option and albums in general:
Basically, all lolisafe-based hosts, which includes mine, has an album feature for registered users, and they can associate uploads to those albums. So with this option, all the uploads you take with the script will then be associated with whichever album you had set your shortcuts into.

**Comment by jschx** [comment]
I would probably leave `-t` out.

**Comment by BobbyWibowo** [comment]
Done :+1:

**Comment by jschx** [comment]
Looks great, thanks a lot!
